components/provider/tests/data_provider.rs

```diff
@@ -225,10 +225,7 @@ fn test_warehouse_owned_dyn_generic() {
 fn test_warehouse_owned_dyn_erased_alt() {
     let warehouse = get_warehouse(DATA);
     let response = get_payload_alt(&warehouse as &dyn ErasedDataProvider);
-    assert!(matches!(
-        response,
-        Err(DataError::UnsupportedResourceKey { .. })
-    ));
+    assert!(matches!(response, Err(DataError::UnsupportedResourceKey { .. })));
```

Comment:
I was getting this diff, too, but running `rustup update` fixes it.

Reply:
_Done._